app/scripts/templates/connect_another_device.mustache

```diff
@@ -64,7 +64,7 @@
       {{/isFirefoxIos}}
     {{/canSignIn}}
     <div class="links">
-      <a href="/connect_another_device/why">{{#t}}Why is this required?{{/t}}</a>
+      <a data-flow-event="link.why" href="/connect_another_device/why">{{#t}}Why is this required?{{/t}}</a>
```

Comment:
I wonder if are overloading the flow event pipeline. I think before we were talking about how flowEvents are only a selected number of events. If we keep adding more and more then queries will be even slower. thoughts on this @philbooth ?